systemtest/src/main/java/io/strimzi/systemtest/resources/jaeger/SetupJaeger.java

```diff
@@ -52,7 +52,7 @@ public class SetupJaeger {
      * Delete Jaeger instance
      */
     private static void deleteJaeger(String yamlContent) {
-        cmdKubeClient().namespace(Environment.TEST_SUITE_NAMESPACE).deleteContent(yamlContent);
+        cmdKubeClient().namespace(JAEGER).deleteContent(yamlContent);
```

Comment:
Why it's `JAEGER` here and `JAEGER_NAMESPACE` on different places?
Also it would be worth mentioning why it's now in different Namespace.

Reply:
It should be `JAEGER_NAMESPACE`, I will change it. I also updated description:

`Move Jaeger deployment from `test-suite-namespace` to `jaeger` namespace. The same pattern is used for cert-manager so it makes sense to me have Jaeger in different namespace as well. Especially when test components are not deployed into `test-suite-namesapce`.`